keyboards/handwired/dactyl_manuform/4x6/keymaps/via/config.h

```diff
@@ -0,0 +1,28 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2023 André Büsgen <andre.buesgen@posteo.de>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Select hand configuration */
+// #define MASTER_LEFT
+// #define MASTER_RIGHT
+
+#define EE_HANDS
```

Comment:
Is there a specific reason for this, or just preference?

Reply:
There is no specific reason except The default keymap uses this setting. 
I stayed as close as possible to the default keymap. 
However, I can change it if another value better aligns with the code style.
Further, I could also refactor the other keymaps according to your suggestions (only if that's helpful).

Reply:
No, it's fine.